go.mod

```diff
@@ -8,9 +8,12 @@ require (
 	github.com/go-openapi/strfmt v0.17.1
 	github.com/go-openapi/swag v0.17.1
 	github.com/go-openapi/validate v0.17.1
+	github.com/google/go-cmp v0.3.1 // indirect
```

Comment:
Is this a transitive dependency?

Reply:
These and the other one were removed when I ran `go mod tidy`, so updating without them